src/services/penaltyDocuments.js

```diff
@@ -68,6 +74,10 @@ export default class PenaltyDocument {
 		});
 	}
 
+	isEmpty(obj) {
+		return JSON.stringify(obj) === JSON.stringify({});
```

Comment:
Just for reference, there's a library called lodash that has a isEmpty method (and plenty of other really useful methods) that does this in a more efficient manner.